src/server.cpp

```diff
@@ -240,6 +240,22 @@ Server::Server(
 
 	if (!gamespec.isValid())
 		throw ServerError("Supplied invalid gamespec");
+
+#if USE_PROMETHEUS
+	m_metrics_backend = std::unique_ptr<MetricsBackend>(new PrometheusMetricsBackend());
+#else
+	m_metrics_backend = std::unique_ptr<MetricsBackend>(new MetricsBackend());
+#endif
+
+	m_uptime_gauge = m_metrics_backend->addGauge("core_server_uptime", "Server uptime (in seconds)");
+	m_player_gauge = m_metrics_backend->addGauge("core_player_number", "Number of connected players");
+	m_step_counter = m_metrics_backend->addCounter("core_server_step_time", "Server step time (in milliseconds)");
+
+	m_timeofday_gauge = m_metrics_backend->addGauge("core_timeofday", "Time of day value");
+	m_lag_gauge = m_metrics_backend->addGauge("core_lag", "Lag value (in seconds)");
```

Comment:
A user isn't going to know what "Lag value" means. The name lag is also completely ambiguous.

```suggestion
	m_lag_gauge = m_metrics_backend->addGauge("core_lag", "The current target server step size (in seconds)");
```

Reply:
what do you suggest here ?

Reply:
"Rough average of recent server step times" ?

Reply:
m_lag is related to dedicated_server_step